src/Command.ts

```diff
@@ -361,17 +361,17 @@ export abstract class Command
      * await int.reply({ ...Command.useButtons(btns), content: "foo" });
      * ```
      */
-    public static useButtons(buttons?: ButtonBuilder | ButtonBuilder[]): { components: ActionRowBuilder<ButtonBuilder>[] } | Record<string, never>
+    public static useButtons(buttons?: ButtonBuilder | Tuple<Tuple<ButtonBuilder, 1|2|3|4|5>, 1|2|3|4|5>): { components: ActionRowBuilder<ButtonBuilder>[] }
     {
-        const actRows = Array.isArray(buttons) ? buttons : (buttons ? [buttons] : undefined);
+        const actRows = Array.isArray(buttons) ? buttons : (buttons ? [[buttons]] : []);
+        const rows: ActionRowBuilder<ButtonBuilder>[] = [];
 
-        if(!actRows || actRows.length === 0)
-            return {};
+        actRows.map(row => {
```

Comment:
just use forEach lol

Reply:
:lowest_quality_pepe: but you're right. a loop makes more sense + the perf impact is negligible since the array is very small
![image](https://user-images.githubusercontent.com/36709480/195361741-77e5e466-b369-44a3-94a6-dd520707df5a.png)